pkg/segment/results/segresults/segresults.go

```diff
@@ -517,7 +517,7 @@ func (sr *SearchResults) GetSegmentStatsResults(skEnc uint16) ([]*structs.Bucket
 		}
 	}
 	aggMeasureResult := []*structs.BucketHolder{bucketHolder}
-	return aggMeasureResult, sr.segStatsResults.measureFunctions, sr.segStatsResults.groupByCols, nil, len(sr.segStatsResults.measureResults)
+	return aggMeasureResult, sr.segStatsResults.measureFunctions, sr.segStatsResults.groupByCols, nil, 1
```

Comment:
Is this change to fix a bug, or was the `len()` always 1?

Reply:
Yes, the `len` will always be 1. That is the bucketCount. And For only `stats` without groupby, the bucket count is always 1.

if you check the [Line #519](https://github.com/siglens/siglens/pull/1774/files/3b46f3aa392361b0c318c597c89067443dfa60d2#diff-153fd9c9d5bb1a446f897390942735ad9010e6656c9fcd80494b41cd5ccc5252R519), We are just assigning one bucket Holder and making it a slice.

And the `len(sr.segStatsResults.measureResults)` gives the length of measure functions like `sum(col1), avg(col2), etc`